### DISCUSSION TITLE
PODMAN 5.2 - Astronomer Setup - Windows - Issue in starting the Podman ( fatal error in javascript)

### DISCUSSION THREAD

**Discussion by ShaMurthy** [podman-desktop/podman-desktop#9143] [General]
Hello Team,

We are trying to use podman for astronomer which is our orchestration tool and we also want to use this for running containerised applications with snowflake.

We are running it on Windows 10 and we are getting error in initializing it.

The error message is mentioned below:

level=fatal msg="error init compose client ssh host connection is not valid: extra path after the host: \"/run/user/1000/podman/podman.sock\""

![image](https://github.com/user-attachments/assets/fc990073-af78-425a-b666-b8aeaac1c683)

We need to address this at the earliest as this is being the major impediment for one of the implementations.

Please assist us in resolving this issue. We would appreciate if you could arrange a short meeting to fix this.

**Comment by jeffmaury** [comment]
Can you join the log from Podman Desktop ? Is compose involved in your setting ?
